<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCallMainPage</class>
 <widget class="QWidget" name="VideoCallMainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>763</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VideoCall</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:'Microsoft YaHei';
 font-size: 12px;
 color:white;
}

#top_widget{
background:#1D2129;
}


#lbl_user_logo{
background-image:url(:img/user_bg);
background-position:center;
background-repeat:no-repeat;
}

#lbl_rec_logo{
background-image:url(:img/red_circle);
background-position:center;
background-repeat:no-repeat;
}

#btn_visible_list{
color:#86909C;
border:none;
background:transparent;
padding-left:16px;
background-image:url(:img/down);
background-position:left;
background-repeat:no-repeat;
}

#line{
background:#4E5969
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="top_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="lbl_expand">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#lbl_expand{
background-image:url(:img/full_demo_logo);
background-position:center;
background-repeat:no-repeat;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="room_info" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#room_info{
position: absolute;
width: 182px;
height: 28px;
left: 629px;
top: 11px;

background: rgba(255, 255, 255, 0.05);
border-radius: 8px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">#label{
font-family: 'Inter';
font-style: normal;
font-weight: 400;
font-size: 12px;
line-height: 22px;
/* identical to box height, or 183% */

text-align: center;

/* Text/2 */

color: #C9CDD4;

opacity: 0.8;
}</string>
           </property>
           <property name="text">
            <string>room_id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_room_id">
           <property name="text">
            <string>1111</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>12</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_time">
           <property name="text">
            <string>00:01</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>230</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1440</width>
       <height>900</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlBarWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#controlBarWidget{
background: #1D2129;
}

QToolButton{
background:transparent;
border-radius: 8px;
font-family: 'Inter';
font-style: normal;
font-weight: 400;
font-size: 12px;
line-height: 22px;
text-align: center;
color: #C9CDD4;
border:none;
}

QToolButton:hover{
background: rgba(255, 255, 255, 0.05);
}

QToolButton:pressed{
background: rgba(255, 255, 255, 0.1);
}

QToolButton:disabled{
 background: rgba(255, 255, 255, 0.3);
}

</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="cameraBtn">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>camera</string>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_camera_off</normaloff>:/img/videocall_camera_off</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="micBtn">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#micBtn{
text-align: left;
}</string>
        </property>
        <property name="text">
         <string>microphone</string>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_mic_off</normaloff>:/img/videocall_mic_off</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="dataBtn">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>real-time_data</string>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_real_time_data</normaloff>:/img/videocall_real_time_data</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="shareBtn">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>screen_sharing</string>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_share</normaloff>:/img/videocall_share</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="beautyBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>beauty</string>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_beauty</normaloff>:/img/videocall_beauty</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="settingBtn">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>set_up</string>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_setting</normaloff>:/img/videocall_setting</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="endCallBtn">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>54</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>end_call</string>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_end_call</normaloff>:/img/videocall_end_call</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/video_call_resource.qrc"/>
 </resources>
 <connections/>
</ui>
